fpgalib/adc.py

```diff
@@ -955,7 +955,7 @@ def registerReadback(self):
         @inlineCallbacks
         def func():
             # build registry packet
-            regs = self.regRun(self.RUN_MODE_REGISTER_READBACK, 0) # 0 reps?
+            regs = self.regRun(self.RUN_MODE_REGISTER_READBACK, {}, 0)
```

Comment:
I think it'd be a good idea to add param names throughout the module. Note that the param names for build1 and build7 are different.